### ISSUE TITLE
List style type.

### ISSUE THREAD

**Issue by c933103** [w3c/jlreq#12]
In css list style type property, it is possible to specify the use of cjk-ideographic, hiragana, hiragana-iroha, katakana, and katakana-iroha as ordered list marker. However, the result of these options as I see from Chrome browser is like this:
![image](https://i.imgur.com/zSl3Xdi.png)
Which have normal western dot behind each characters. As I understand, this is not a common way to write this type of list in Japanese. Should it be specified in jlreq that this sort of lists usually use the symbol "、" instead? And perhaps also need to contact various browser vendors and such for this?

**Comment by upsuper** [comment]
According to [CSS Counter Styles Level 3](https://drafts.csswg.org/css-counter-styles/#simple-alphabetic), they should use "、" as suffix, and Firefox has implemented that. You should probably just file bugs to browsers which don't do this.

**Comment by c933103** [comment]
Ah I see. But as jlreq is about Japanese layout that is not limited to those that are using css and browser webpage, maybe it would still be beneficial to include this in jlreq?

**Comment by r12a** [comment]
I agree that jlreq should say something about counter styles. We don't have immediate plans to update the document, but i'll leave this open as a reminder until we do.

**Comment by r12a** [comment]
By the way, we do have tests and results for all the counter styles in the spec, and more in our Custom Counter Styles doc, although only the former post the a result for whether the symbol was what was expected.

See 
https://www.w3.org/International/tests/repo/results/counter-styles
https://www.w3.org/International/tests/repo/results/predefined-counter-styles and
https://www.w3.org/International/tests/repo/results/predefined-counter-styles

**Comment by himorin** [comment]
also in #107.
